src/solution.jl

```diff
@@ -85,7 +85,9 @@ function solve!(
     if problem_vexity(p) in (ConcaveVexity(), NotDcp())
         throw(DCPViolationError())
     end
-    context = Context(p, optimizer_factory)
+    context, formulation_time, formulation_bytes =
+        @timed Context(p, optimizer_factory)
+    # @show (formulation_time, Base.format_bytes(formulation_bytes))
```

Comment:
we can revert this, but it would kinda be nice to be able to get this info... maybe we can stick it in the solution struct to query later?

Reply:
What about printing it if `!silent_solver`?

I do like that cvxpy gives you information on it's compilation process.